src/components/Table/GridTable.stories.tsx

```diff
@@ -134,6 +134,128 @@ export function VirtualFiltering() {
   );
 }
 
+export function VirtualFilteringWithFilterablePin() {
+  const rows: GridDataRow<Row>[] = useMemo(
+    () => [
+      simpleHeader,
+      ...zeroTo(1_000).map((i) => ({
+        kind: "data" as const,
+        id: String(i),
+        data: { name: `ccc ${i}`, value: i },
+        pin: i === 500 ? { at: "first" as const, filter: true } : undefined,
```

Comment:
Afaict both this line and line 231 are using `filter: true`, and I was expecting to see a `filter: false`...

Oh, is the difference that one story is virtual and the other is not?

I dunno, each of these stories at quite a few LOC and, with the tests, I'm not sure we really need three of them, or really any of them?

Like if we _really_ needed a story for filtering, it would be much simpler to have a small story with like three rows, where one row had `filter: true`, one row had `fliter: false`, and the 3rd had no filter. Or something like that.

As-is it seems like we're copy/pasting already-pretty-complicated stories and making them even more complicated. We should try to simplify and focus on just as small as use case as possible.

That said, I think let's just skip stories for this change all together, and assume the unit tests will cover it.

Reply:
okay, i think we can leave one story then, with 3 types of pins, at least as an example for people to be aware of the differences